daemon/src/peer/run_state.rs

```diff
@@ -317,7 +317,9 @@ impl RunState {
                         provider: PeerInfo { peer_id, .. },
                         result,
                     } => {
-                        cmds.extend(self.waiting_room.found(&urn, peer_id, SystemTime::now()));
+                        if self.waiting_room.get(&urn).is_some() {
+                            cmds.extend(self.waiting_room.found(&urn, peer_id, SystemTime::now()));
+                        }
 
                         if let PutResult::Applied(_) = result {
```

Comment:
Can it happen that the urn is not in the waiting room, but there is still an include file which may need updating?

Reply:
Once you've replicated a project, the include will still need updating. Say if the delegates change and thus the remotes will need changing.